lib/url.js

```diff
@@ -87,7 +87,7 @@ function urlParse(url, parseQueryString, slashesDenoteHost) {
 
 Url.prototype.parse = function(url, parseQueryString, slashesDenoteHost) {
   if (typeof url !== 'string') {
-    throw new TypeError("Parameter 'url' must be a string, not " + typeof url);
+    throw new TypeError('Parameter "url" must be a string, not ' + typeof url);
```

Comment:
Why use double quotes in one place, and single quotes in the other? Also, I think a closing quote is missing.